test/requirements_csv.py

```diff
@@ -0,0 +1,68 @@
+# Copyright (C) 2021 Intel Corporation
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+# Script to add versions to requirements.csv
+
+import csv
+import os
+import re
+from importlib.metadata import version
+
+REQ_CSV = os.path.abspath(
+    os.path.join(os.path.dirname(__file__), "..", "requirements.csv")
+)
+DOC_CSV = os.path.abspath(
+    os.path.join(os.path.dirname(__file__), "..", "doc", "requirements.csv")
+)
+CACHE_CSV = os.path.join(
+    os.path.expanduser("~"), ".cache", "cve-bin-tool", "requirements.csv"
+)
+HTML_DEP_PATH = os.path.abspath(
+    os.path.join(
+        os.path.dirname(__file__),
+        "..",
+        "cve_bin_tool",
+        "output_engine",
+        "html_reports",
+        "js",
+    )
+)
+HTML_DEP_CSV = os.path.join(HTML_DEP_PATH, "dependencies.csv")
+
+
+def get_cache_csv_data(file):
```

Comment:
Let's put this into a pytest testcase. let's also look at offending packages, and have an allowable list. Which will contain jquery and reportlab. Then we can create issues to track and fix.

Reply:
I'm a bit confused about it 😅 
I have added the allowable list and wrapped most of the script into a testcase.
Should I run the CVE check inside the script for assertion?
Also should I rename the file by adding a `test_` in front of the file?